script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -1099,7 +1103,8 @@ The true identity of our enemy.
 [sha:31f10d90c2426ae4c9a13970e74b8efebae85563]{
 -- Page 371, Offset 25203.
 -- そいつがどんなヤツなのか、ここで聞いておくべきだ。
-I should use this chance to find out what kind of person he is.
+// Should we keep pronouns neutral until Shiki knows who Roa is? It's a little more accurate but also reads a little awkwardly in places.
```

Comment:
I think that Arc can talk about Roa as male from the get-go, since she knows him and that he is originally/currently incarnated as a dude. After the first time she refers to him as 'he' it's probably safe for Shiki to as well, which looks like it would make this the only they?